schemas.py

```diff
@@ -0,0 +1,37 @@
+import datetime
+
+from pydantic import BaseModel
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str
+
+    class Config:
+        orm_mode = True
```

Comment:
Are u sure that every schema that inherits from this base class needs this option?
In general, you should use it on demand, since when `orm_mode=True` pydantic will use a less efficient algorithm when performing actions